src/darwin/Framework/CHIP/MTRDevice_Concrete.mm

```diff
@@ -3922,7 +3952,7 @@ - (void)_noteDataVersion:(NSNumber *)dataVersion forClusterPath:(MTRClusterPath
         clusterData = [[MTRDeviceClusterData alloc] initWithDataVersion:dataVersion attributes:nil];
```

Comment:
This needs to set the pending data version, not the current one, right?

Reply:
Thanks for catching this, which is actually the common case for very first report. I've made the change